ptm_python/ptm_tools.py

```diff
@@ -74,13 +72,134 @@ def fromRigidity(cls, rigidity_GV):
 
 class Proton(Particle):
     charge = 1
-    mass = 1.00727647  # AMU
-    restmass = 938.27231  # MeV
+    mass, _, _ = constants.physical_constants['proton mass in u']  # AMU
+    restmass, _, _ = constants.physical_constants['proton mass energy equivalent in MeV']
+
     def __init__(self, energy):
         self.energy = energy
         super().__init__()
 
 
+class StormerCutoff():
+    def __init__(self, cd_moment=29950.126):
+        """
+        Parameters
+        ==========
+        cd_moment : float
+            Centered dipole moment of epoch in nT. Can, for example, be obtained
+            from spacepy.igrf (after instantiation, the value is in the moment
+            attribute). Default is the dipole moment for the 2010 epoch.
+        """
+        self.set_coefficient(cd_moment)
+
+    def set_coefficient(self, cd_moment):
+        re_cm = 6371.0008*1e5  # Volumetric radius in cm
```

Comment:
Might be worthwhile moving the volumetric radius to the constants, just so all code that might use it is grabbing it from the same place

Reply:
Good point. I'll update that soon. I'd also expanded the docstrings slightly so I'll make sure I have references in there and then push.